src/bundles/analytics.test.js

```diff
@@ -30,18 +30,19 @@ function createStore (analyticsOpts = {}) {
   )()
 }
 
-it('should disable analytics by default', () => {
+it('should enable analytics by default', () => {
   const store = createStore()
-  expect(store.selectAnalyticsEnabled()).toBe(false)
-  // no events will be sent as no consents have been given
-  expect(store.selectAnalyticsConsent()).toEqual([])
-  // user has not explicitly opted in or out yet
-  expect(global.Countly.opt_in).not.toHaveBeenCalled()
-  expect(global.Countly.opt_out).not.toHaveBeenCalled()
+  expect(store.selectAnalyticsEnabled() || store.selectAnalyticsInitialConsent()).toBe(true)
```

Comment:
we should explicitly test each of these. If store has X val, analyticsEnabled should be true, initialConsent should be true.. if store has Y val, one true one false.. etc.